src/sc_data/data.py

```
@@ -212,66 +288,77 @@ def update(self):
+                    self.cache_db_path,
+                )
                 return True
             else:
-                # Cache write failed, but we might still be able to use the embedded DB
-                logger.warning("Cache write failed, using existing database")
+                logger.warning(
+                    "update: cache write failed after %.1fs", time.time() - t0
+                )
                 return False
 
         except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)